README.md

```diff
@@ -80,6 +80,11 @@ this decreases the compile and link time from ~10 seconds to ~4 seconds.
 Install packages and init repository:
 
 ```sh
+=======
+Install Packages and Init Repository
+
+```bash
```

Comment:
I think if you add this here, you'll need to remove the `sh` on line 82.

Reply:
Not sure how that made its way in there, should be even with master now